src/pspm_filtfilt.m

```diff
@@ -70,6 +70,9 @@
 if len <= nfact
     error('Data must have length more than 3 times filter order.');
 end
+if na == 1
+    warning('ID:invalid_input','The filter parameters ''a'' must be of dimension 2.'); return;
+end

```

Comment:
At this point it is possible that length of `a` has increased due to line 66. Therefore even if `na == 1`, it is possible that the code below would work. I think it would be better if we rather verify that `nfilt` is at least 2, which corresponds to having a filter of order at least 1. If it doesn't hold then we can emit a warning, saying that the parameter vectors are too short.

Reply:
So after the discussion in #58, we want to check if `nfilt == 1`, and return the signal unchanged if it is. All the other cases seem appropriately handled to me. What do you think?

Reply:
Yes for me too. I changed and merged that.